prestoadmin/prestoclient.py

```diff
@@ -129,10 +137,12 @@ def execute_query(self, sql, schema="default", catalog="hive"):
 
             self.response_from_server = json.loads(answer)
             _LOGGER.info("Query executed successfully: %s" % (sql))
+            _LOGGER.info("Using LDAP = %s" % self.coordinator_config.use_ldap())
```

Comment:
I propose moving this above where we make the request and including the username. LIke maybe around 124?